client/my-sites/stats/post-performance/index.jsx

```diff
@@ -189,9 +129,12 @@ export default connect( ( state, ownProps ) => {
 	const query = { status: 'published' };
 	const posts = site ? getSitePostsForQuery( state, site.ID, query ) : null;
 	const post = posts && posts.length ? posts[ 0 ] : null;
+	const countViews = post && site ? getPostStat( state, 'views', site.ID, post.ID ) : null;
```

Comment:
`viewCount` _feels_ like a better variable/prop name here.  `countViews` almost sounds like an action to count the number of views of the component - though obviously that would never happen.

Just struck me as odd the first time I read the name in the proptype declaration above 😆